src/tactic/interactive_expr.lean

```diff
@@ -153,6 +161,12 @@ meta def view {γ} (tooltip_component : tc subexpr (action γ)) (click_address :
     on_click (λ _, action.on_click ca),
     key s
   ] [html.of_string s]]
+| ca (sf.block i a) := do
+  inner ← view ca a,
+  pure [h "span" [cn "indent-code", style [("--indent-level", to_string i)]] inner]
+| ca (sf.highlight c a) := do
+  inner ← view ca a,
+  pure [h "span" [cn $ "highlight_" ++ c.to_string] inner]
```

Comment:
This should *probably* be translated to one of the classes like `hot-pink` that are already in tachyon.

Reply:
I'd rather not do that. Because it encodes a hard mapping between mathlib's source code and the plugin, whereas the CSS class is conceptually much more flexible. Plus highlighting might also be encoded with other means than color like italics or bold. From an accessibility point-of-view, I'd rather have the CSS handle that, if it's ok.

Reply:
Everything about the lean widget system already assumes a hard mapping, namely it assumes that a CSS library called Tachyons is being used. So any styling choices should be made in the lean code not in a separate CSS file (I admit `indent-level` is an exception).  I don't think it makes sense to have `color.red` mean anything other than colouring the text red.

Reply:
I guess it's awkward for people who have a kind of colourblindness where they can't distinguish certain colours from black or white (idk if that's possible). I don't think we will have cases where it is necessary for the user to be able to distinguish the highlight colours to understand the format string.